clientapi/routing/routing.go

```diff
@@ -410,4 +410,25 @@ func Setup(
 			}}
 		}),
 	).Methods(http.MethodGet, http.MethodOptions)
+
+	r0mux.Handle("/user/{userId}/rooms/{roomId}/tags",
+		common.MakeAuthAPI("get_tag", authData, func(req *http.Request, device *authtypes.Device) util.JSONResponse {
+			vars := mux.Vars(req)
```

Comment:
```suggestion
			vars, err := common.URLDecodeMapValues(mux.Vars(req))
			if err != nil {
				return util.ErrorResponse(err)
			}
```
These will need to change because of https://github.com/matrix-org/dendrite/commit/33a13925417612b032c1cbbb8ee62eb053faa5f8